### ISSUE TITLE
RSS Feed

### ISSUE THREAD

**Issue by pedrommone** [valeriansaliou/vigil#4]
I usually use the FEED (as a user) to monitor 3rd party servers on Slack!

**Comment by valeriansaliou** [comment]
So, what would you expect in the RSS feed? Do you have any XML example?

**Comment by pedrommone** [comment]
@valeriansaliou here is the StatusPage sample: https://status.bitrise.io/history.rss

At Slack, I just setup a "RSS Feed addon" and the message show this way

![https://i.imgur.com/2KBdBRM.png](https://i.imgur.com/2KBdBRM.png)

**Comment by valeriansaliou** [comment]
Thanks. Did you know you could already link Vigil to Slack for notifications? Would be directly equivalent to using a RSS feed, with the benefit of getting a real-time notification.

**Comment by pedrommone** [comment]
@valeriansaliou yes I know, but it works *only* for inside usage, right? I want to monitor 3rd party softwares that we rely on. To be more specific, Crisp.chat.

**Comment by valeriansaliou** [comment]
You'd rather not monitor Crisp as we usually have 4-5 status changes in the same day, for a normal day. That can be some systems slowing down or stuff we work on deploying which don't affect the actual service from an user point of view, but still triggers Vigil.

**Comment by pedrommone** [comment]
Oh, sure.

Is there a way to monitor *major* issues?

**Comment by valeriansaliou** [comment]
Nope.
